plugin.video.hanssettings/addon.py

```diff
@@ -0,0 +1,155 @@
+'''
+    Hans Settings
+    ~~~~~~~
+
+    An XBMC addon for watching hanssettings 
+    :license: GPLv3, see LICENSE.txt for more details.
+    
+'''
+import sys
+try:
```

Comment:
you can use sys.version_info to check if the interpreter is python 2 or python3 (better than try/except)

Reply:
Done